website/docs/API.md

```diff
@@ -552,7 +552,7 @@ Defined as:
 ```ts
 function waitForElementToBeRemoved<T>(
   expectation: () => T,
-  { timeout: number = 4500, interval: number = 50 }
```

Comment:
This seemed like a mistake based on what is in code (?)

Reply:
Indeed it is, thanks for fixing that!